backend/deployment_triggers/dataall_migrate_handler.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import os
+from migrations.dataall_migrations.herder import Herder
+import boto3
+
+logger = logging.getLogger()
+logger.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
+
+client = boto3.client('ssm')
+
+envname = os.environ.get('ENVNAME', 'local')
+PARAM_KEY = (f'/dataall/{envname}/dataall-migration/revision',)
+
+
+def get_parameter_from_parameter_store():
```

Comment:
Did you consider storing the revision in the db?

Reply:
Yes, I was thinking about it. But then I thought: 
1. It's only one parameter and it's related to AWS resources
2. Dataall migrations are not related to DB in general, so, in general it shouldn't access DB
3. SSP Parameter store is free

So, I went for SSM

Reply:
would having it in SSM make local development trickier?

Reply:
also I am thinking that RDS is the only place we keep the app's state but with this change we will also use SSM. I am trying to over think it because the decision we make now might be difficult to be changed in the future.

Reply:
They are very similar options, for me the "untie" factor is that if we store it in db we can run the migrations locally without a manually-created SSM parameter. On the other hand, I think most migrations will have to look-up information in RDS

Reply:
@petrkalos 
Basically, this "dataall-migrations" are designed for software changes, that are not related to database -- so, they are AWS-related. It means, that anyhow we need to have aws-access to run them, can't run them locally.

Reply:
Kind of, they are AWS operations on data.all deployed infra or other SDK calls (e.g. shares) that can be deployed/executed locally. I would like to have a running deployment of data.all locally run the migrations and see if the changes are correct.